### DISCUSSION TITLE
Areas

### DISCUSSION THREAD

**Discussion by kuduacz** [dwainscheeren/dwains-lovelace-dashboard#494] [Q&A]
How can i choose sensor to show in my areas:
![image](https://user-images.githubusercontent.com/77231628/165931434-8a88f393-8c4b-4f7d-8657-10473acae3a1.png)
i mean i wont to choose temperature sensor to show here, and how to disable to show lock? (its lock from humidifier so i dont want then to show here).

**Comment by dwainscheeren** [comment]
Disable the entity in DD inside the area

**Reply by kuduacz** [reply]
Ok, Ty a lot.
but:
it is possible to change the color of people outside the house from gray to another?
Is it possible to select entities that will be displayed in "areas" such as movement, window / door, thermostat or not?
Ability to change the displayed elements in the top bar (people, movement, doors, windows, etc.
It is generally about being customizable.

**Reply by dwainscheeren** [reply]
Changing the gray color is currently not possible please add it in the feature requests discussion. The other questions are all possible just edit the entities in ha itself, set a door for a binary sensor as show as etc. DD picks this up and automatic does everything.

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
